qa/tasks/mgr/test_dashboard.py

```diff
@@ -48,12 +45,7 @@ def test_urls(self):
         # the content is correct or even renders!
 
         urls = [
-            "/health",
-            "/servers",
-            "/osd/",
-            "/osd/perf/0",
-            "/rbd_mirroring",
-            "/rbd_iscsi"
+            "/",
```

Comment:
Actually, we should also test our controllers in the smoke test. But that's a different PR

Reply:
Is this something you feel should be addressed in this PR?